base-theme/layouts/shortcodes/sub.html

```diff
@@ -1 +1 @@
-<sub>{{ .Get 0 | markdownify }}</sub>
+<sub>{{- .Get 0 | markdownify -}}</sub>{{- "" -}}
```

Comment:
to me, this `{{- "" -}}` looks like a hacky solution but I couldn't think of any way to get rid of this extra-whitespace caused by the `<sub>` element

Reply:
Interesting. It seems like another option is to remove the end-of-line at the end of the file, like https://github.com/mitodl/ocw-hugo-themes/commit/6b3a9f8505f39290f0521b55afc5cc2c051b8ed4

But considering that (a) our convention seems to be all files should end in EOLs, and (b) lots of editors add them automatically, relying on shortcodes **not** having EOLs at end of file seems like it might just cause problems down the road.